apps/common-app/src/apps/reanimated/examples/IPodExample.tsx

```diff
@@ -73,30 +71,21 @@ export default function IPodExample() {
     },
   });
 
-  type Vector2D = {
-    x: number;
-    y: number;
-  };
-  type AnimatedGHContext = {
-    start: Vector2D;
-    last: Vector2D;
-  };
-  const gestureHandler = useAnimatedGestureHandler<
-    PanGestureHandlerGestureEvent,
-    AnimatedGHContext
-  >({
-    onStart: (e, ctx) => {
-      ctx.start = { x: e.x, y: e.y };
-      ctx.last = ctx.start;
-    },
-    onActive: (e, ctx) => {
+  const start = useSharedValue({ x: 0, y: 0 });
+  const last = useSharedValue({ x: 0, y: 0 });
+
+  const gesture = Gesture.Pan()
+    .onBegin((e) => {
+      start.value = { x: e.x, y: e.y };
+      last.value = { x: e.x, y: e.y };
+    })
+    .onUpdate((e) => {
       const currentPoz = { x: e.x, y: e.y };
```

Comment:
```suggestion
      const currentPos = { x: e.x, y: e.y };
```

Maybe use proper english name here and change `z` to `s` if you are already updating these examples?